kong/plugins/hmac-auth/access.lua

```diff
@@ -129,6 +134,22 @@ local function validate_clock_skew(headers, date_header_name, allowed_clock_skew
   return true
 end
 
+local function validate_request_body(headers, body)
+  local sha_recieved = headers[DIGEST]
+
+  if body == nil then
+    return true
```

Comment:
style, needs a blank line after return. not a blocker, we can adjust this as part of merge